tests/system/common.py

```diff
@@ -819,3 +822,26 @@ def kill_process_on_host(hostname, pattern):
     else:
         print("Killed no pids")
     return pids
+
+
+def service_available_predicate(service_name):
```

Comment:
Can you also add 1-2 lines of documentation?

Reply:
sure!